README.md

```diff
@@ -4,37 +4,100 @@
 <https://github.com/asecurityteam/nexpose-asset-attributor>
 
 <!-- TOC -->
-
 - [nexpose-asset-attributor - Attribute Assets from Nexpose to Teams and People](#nexpose-asset-attributor)
     - [Overview](#overview)
     - [Quick Start](#quick-start)
     - [Configuration](#configuration)
+        - [Logging](#logging)
+        - [Stats](#stats)
     - [Status](#status)
     - [Contributing](#contributing)
         - [Building And Testing](#building-and-testing)
         - [Quality Gates](#quality-gates)
         - [License](#license)
         - [Contributing Agreement](#contributing-agreement)
-
 <!-- /TOC -->
 
 <a id="markdown-overview" name="overview"></a>
 ## Overview
 
-<What does this project do?>
-<What does this project _not_ do?>
-<Why did we make this project?>
-<Links to other references or material.>
+The nexpose-asset-attributor provides a lambda handler with accepts payloads containing an asset and a list of
```

Comment:
nit/question: did you mean with -> which

Reply:
Yes